src/Pages/2DPrinting/2DPrinting.js

```diff
@@ -160,191 +236,133 @@ export default function Printing(props) {
-                      />
-                      <Input
-                        type='text'
-                        disabled={!pages}
-                        placeholder='1-5, 7, 9-11'
-                        onChange={async e => {
-                          setPageRanges(e.target.value);
-                          const x = await parseRange(
-                            e.target.value, numPages
-                          );
-                          setUsedPages(x);
-                          handleCanPrint(x, copies);
-                        }}
-                      />
-                    </Label>
-                  </FormGroup>
+                  </Label>
+                </FormGroup>
+                <FormGroup check>
+                  <Label check>
+                    <Input
+                      checked={pages}
+                      onChange={() => {
+                        setPages(true);
+                        handleCanPrint(usedPages, copies);
+                        setLoadPreview(true);
+                      }}
+                      {...pagesInputProps}
+                    />
+                    <Input
+                      type='text'
+                      disabled={!pages}
+                      placeholder='1-5, 7, 9-11'
+                      onChange={async e => {
+                        setPageRanges(e.target.value);
+                        const x = await parseRange(
+                          e.target.value, numPages
+                        );
+                        setUsedPages(x);
+                        handleCanPrint(x, copies);
+                        setLoadPreview(true);
+                      }}
+                    />
+                  </Label>
                 </FormGroup>
-                <Label className='center-blocks'>
-                  Note: All prints are black ink only
-                </Label>
-                <br />
-              </div>
+              </FormGroup>
+              <Label className='center-blocks'>
+                Note: All prints are black ink only
+              </Label>
+              <br />
             </Col>
           </Row>
         </ModalBody>
         <ModalFooter>
-          <Button
-            color='danger'
-            style={{
-              float: 'left'
-            }}
-            onClick={() => {
-              setPreviewModal(false);
-              setPages(false);
-              let arr = new Set(range(1, numPages + 1));
-              setUsedPages(arr);
-            }}
-          >
-            Back
-          </Button>
-          <Button
-            color='success'
-            style={{
-              float: 'right'
-            }}
-            onClick={() => {
-              setConfirmModal(!confirmModal);
-            }}
-            disabled={!canPrint}
-          >
-            Print!
-          </Button>
+          <Button {...backButtonProps}> Back </Button>
+          <Button {...printButtonProps}> Print! </Button>
```

Comment:
Is there a way we can tell the user the print was successful or not? as of now it goes quiet after we submit a request.